src/main/java/com/gdschongik/gdsc/domain/member/domain/Member.java

```diff
@@ -249,6 +250,10 @@ public void updateMemberInfo(
     }
 
     public void completeUnivEmailVerification(String univEmail) {
+        // 이미 인증되어있으면 에러
+        if (associateRequirement.isUnivVerified()) {
+            throw new CustomException(ErrorCode.EMAIL_ALREADY_VERIFIED);
```

Comment:
line 4 보시면 import static 되어있습니다~

```suggestion
            throw new CustomException(EMAIL_ALREADY_VERIFIED);
```